docs/protocol.md

```diff
@@ -545,7 +545,7 @@ HTTP/1.1 200 OK
     {
       "id": "A string no longer than 7 characters.",
       "type": "Secp256k1VerificationKey2018 | RsaVerificationKey2018",
-      "publicKeyJwk": "Must be JWK format for Secp256k1VerificationKey2018, else another property can be used.",
+      "jwk": "Must be JWK format for Secp256k1VerificationKey2018, else another property can be used.",
```

Comment:
This is not valid.... according to the did core spec... either open a PR to get this property added to did core, or use `publicKeyJwk`... see also: https://github.com/decentralized-identity/sidetree/issues/539

Reply:
This representation is not surfaced externally, it will still be `publicKeyJwk` in DID document when resolution occurs.